svf-llvm/include/SVF-LLVM/LLVMModule.h

```diff
@@ -323,6 +326,15 @@ class LLVMModuleSet
         return it->second;
     }
 
+    std::vector<std::string> getFunctionAnnotations(const Function* fun)
```

Comment:
This function has a redundant vector copy. Consider return a reference to the vector, return an empty field `emptyAnnotations` if not found. Nevertheless, I think a better solution is to delete this function...